integration-tests/test/test_internal.py

```diff
@@ -124,7 +124,7 @@ def test_parse_show_block_output() -> None:
 |\n        @{Unforgeable(0x10d8ba59a837e8cd2ca8b29d37125669c3af1260c46949e326bde792f9272d0d)}!((x2 - x0)) ;\n        false => @{x1}!(false) |\n        @{Unforgeable(0x10d8ba59a837e8cd2ca8b29d37125669c3af1260c46949e326bde792f9272d0d)}!(x2)\n      }\n    } ;\n    false => @{x1}!(false)\n  }\n} |\nfor( @{\"add\"}, @{x0}, @{x1} <= @{Unforgeable(0x0f851a3a7f2de4323a43a45d466a7aba1c24a1d368946c9cf267eb13a4d0016a)} ) {\n  match (x0 >= 0) {\n    true => for( @{x2} <- @{Unforgeable(0x10d8ba59a837e8cd2ca8b29d37125669c3af1260c46949e326bde792f9272d0d)} ) {\n      match ((x2 + x0) > x2) {\n        true => @{x1}!(true) |\n        @{Unforgeable(0x10d8ba59a837e8cd2ca8b29d37125669c3af1260c46949e326bde792f9272d0d)}!((x2 + x0)) ;\n        false => @{x1}!(false) |\n        @{Unforgeable(0x10d8ba59a837e8cd2ca8b29d37125669c3af1260c46949e326bde792f9272d0d)}!(x2)\n      }\n    } ;\n    false => @{x1}!(false)\n  }\n} |\nfor( @{\"value\"}, @{x0} <= @{Unforgeable(0x0f851a3a7f2de4323a43a45d466a7aba1c24a1d368946c9cf267eb13a4d0016a)} ) {\n  for( @{x1} <- @{Unforgeable(0x10d8ba59a837e8cd2ca8b29d37125669c3af1260c46949e326bde792f9272d0d)} ) {\n    @{x0}!(x1) |\n    @{Unforgeable(0x10d8ba59a837e8cd2ca8b29d37125669c3af1260c46949e326bde792f9272d0d)}!(x1)\n  }\n} |\nfor( @{x0}, @{x1}, @{x2}, @{x3}, @{x4} <= @{Unforgeable(0xb9a5addc47c459f22ab562af180ae6bc10799956f2c9d1b1dc531feb2d8c8a18)} ) {\n  new x5, x6, x7 in {\n    for( @{x8} <- @{Unforgeable(0x059b9866df5308ec4ae6e8799bc0fcc79e81222555c16833180beca29f57405d)} ; @{x9} <- @{Unforgeable(0x4de349932e7e2443526ad0a4923cd69c0027184659a6c475112fecca2008dedb)} ) {\n      @{Unforgeable(0x39bdf789fb12e1c3487abce560193e3a9545d2d7e311aca70f6f47d36ba5115a)}!((x9).size(), x8, *x5) |\n      for( @{x10} <- x5 ) {\n        x6!(x9, x10, {}, *x7) |\n        for( @{x11} <- x7 ) {\n          @{x4}!((true, \"Bond successful!\")) |\n          @{Unforgeable(0x4de349932e7e2443526ad0a4923cd69c0027184659a6c475112fecca2008dedb)}!((x11).set(x0,((x2 - x8), x1, x3, ((x9).size() + 1))))\n        }\n      }\n    } |\n    for( @{x8}, @{x9}, @{x10}, @{x11} <= x6 ) {\n      match x8 {\n        {x13 : (x14, x15, x16, x17)...x12} => x6!(x12, x9, (x10).set(x13,((x14 + (x9).nth((x17 - 1))), x15, x16, x17)), x11) ;\n        _ => @{x11}!(x10)\n      }\n    }\n  }\n} |\nfor( @{x0}, @{x1}, @{x2} <= @{Unforgeable(0x39bdf789fb12e1c3487abce560193e3a9545d2d7e311aca70f6f47d36ba5115a)} ) {\n  new x3, x4, x5, x6, x7, x8 in {\n    @{bundle+ {   Unforgeable(0x6df7905a81f51e556695f3cbeccd1a81ffa1f3ab4c0884d18324d709cc8ea406) }}!(\"range\", 1, (x0 + 1), *x3) |\n    for( @{x9} <- x3 ) {\n      @{bundle+ {   Unforgeable(0x6df7905a81f51e556695f3cbeccd1a81ffa1f3ab4c0884d18324d709cc8ea406) }}!(\"map\", x9, *x4, *x5) |\n      for( @{x10} <- x5 ) {\n        @{bundle+ {   Unforgeable(0x6df7905a81f51e556695f3cbeccd1a81ffa1f3ab4c0884d18324d709cc8ea406) }}!(\"fold\", x10, 0, *x6, *x8) |\n        for( @{x11} <- x8 ) {\n          @{x2}!(([(((x10).nth(0) + x1) - x11)] ++ (x10).slice(1,x0)))\n  
```

Comment:
can you explain the changes in this file?

Reply:
These are internal tests for the integration test framework; the values shouldn't matter